app/config/config.yml

```diff
@@ -4,145 +4,144 @@ imports:
   - resource: services.yml
 
 parameters:
-    api_connect_timeout: 2
-    api_timeout: 5
-    api_connect_timeout_slow: 3
-    api_timeout_slow: 10
-    honeypot_field: email_address
-    mustache_stat_props: [size, mtime]
+  api_connect_timeout: 2
```

Comment:
Modify this file so we have 4 space indentation so that this diff reduces to the few changes that you made

Reply:
This may have been the Prettier plugin that I had installed from the course I did. I have disabled it now